### PR TITLE
Added PID to server list.

### PR DESCRIPTION
**Pull request by 1stian**
Hello, I've added PID to the server list.. So if you want, take a look. If I remember correctly I've read it that someone asked for it in suggestions... But I would also like to show the PIDs in the list.

### CODE REVIEW

**Comment by 1stian** [comment]
wGSM_PID

File: WindowsGSM/MainWindow.xaml.cs [hunk 1]
```diff
@@ -2656,9 +2664,16 @@ private async Task EndAllRunningProcess(string serverId)
             });
         }
 
-        private void SetServerStatus(ServerTable server, string status)
+        private void SetServerStatus(ServerTable server, string status, string pid)
```
Comment (BattlefieldDuck) [inline WindowsGSM/MainWindow.xaml.cs:2667]: how about this? 👍 
```cs
private void SetServerStatus(ServerTable server, string status, string pid = null)
```

Comment (1stian) [reply WindowsGSM/MainWindow.xaml.cs]: Yea, that would be much better! Didn't think about that.. At all.

**Comment by BattlefieldDuck** [comment]
big thanks, but can you also fix the if else and try catch block coding style? I want to keep in C# style in this project.

**Comment by 1stian** [comment]
Yea, I will. I just noticed that myself, the formatting changed. That was not my intention. Will fix!

**Comment by 1stian** [comment]
Fixed btw ;)

**Pull request merged**